src/components/settings/index.jsx

```diff
@@ -14,17 +16,18 @@ import {
   SETTINGS_PARAM_MATCH,
 } from './data/constants';
 import SettingsTabs from './SettingsTabs';
+import SettingsContext from './SettingsContext';
 
 const PAGE_TILE = 'Settings';
 
 /**
  * Behaves as the router for settings page
  * When browsing to {path} (../admin/settings) redirect to default tab
  */
-const SettingsPage = () => {
+const SettingsPage = ({ enterpriseId }) => {
   const { path } = useRouteMatch();
   return (
-    <>
+    <SettingsContext enterpriseId={enterpriseId}>
```

Comment:
nit: Curious whether `SettingsPage` should be connected to the Redux store to grab the `enterpriseId` if this component solely uses it to pass it to `SettingsContext`? Or should `SettingsContext` be the component that's connected to the Redux store to grab the `enterpriseId` and avoid prop drilling here?